docs/dev/development.adoc

```diff
@@ -29,7 +29,8 @@ First setup your fork of the odo project, following the steps below
 
  . Clone your fork:
 +
-NOTE: The following commands assume that you have the `$GOPATH` environment variable properly set. We highly recommend you place `odo` code into the $GOPATH.
+NOTE: odo uses `go modules` to manage dependencies which means you can clone the code anywhere you like but for backwards compatibility
```

Comment:
I thought it was enough to be using go.mod. What's the backwards compatibility that we're talking about here?

Reply:
If I remember correctly - go version 1.13 introduced go mod - if someone is using older version of golang then they won’t have access to golang modules